WordPress/Classes/ViewRelated/NUX/SigninHelpers.swift

```diff
@@ -39,8 +38,8 @@ import Mixpanel
     class func showLoginFromPresenter(_ presenter: UIViewController, animated: Bool, thenEditor: Bool) {
         let storyboard = UIStoryboard(name: "Login", bundle: nil)
         if let controller = storyboard.instantiateInitialViewController() {
-            let navController = NUXNavigationController(rootViewController: controller)
-            presenter.present(navController, animated: animated, completion: nil)
+            //let navController = NUXNavigationController(rootViewController: controller)
```

Comment:
Maybe remove the commented line and add a note that the initial view controller is a `NUXNavigationController` so its clear for anyone who sees `WordPressAppDelegate isWelcomeScreenVisible` and wonders about it.